src/main/java/com/stripe/model/ExpandableField.java

```diff
@@ -0,0 +1,35 @@
+package com.stripe.model;
+
+public class ExpandableField<T> {
```

Comment:
Could you add a basic comment describing what this class is for? Possibly with a link back to:

https://stripe.com/docs/api#expanding_objects